timepiece/crm/views.py

```diff
@@ -3084,3 +3085,36 @@ def project_download_attachment(request, project_id, attachment_id):
         return HttpResponseRedirect(project_attachment.get_download_url())
     except:
         return HttpResponse('Project attachment could not be found.')
+
+def get_minding(request):
+    user = request.user
+    projects = Project.objects.filter(
+        point_person=user, status=4)[:3]
```

Comment:
suggest you change this so from `status=4` to `get_setting('TIMEPIECE_DEFAULT_PROJECT_STATUS')'`.  If it is not at the top of the file, you get can `get_setting` with `from timepiece.utils import get_setting`.